source/installguide/configuration.rst

```diff
@@ -1460,6 +1460,121 @@ storpool.volume.tags.checkup      Minimal interval (in seconds) to check and rep
 storpool.snapshot.tags.checkup    Minimal interval (in seconds) to check and report if a StorPool Snapshot created by CloudStack exists in CloudStack's database
 ================================= ====================================================================================================================================================================
 
+HPE Primera/3PAR Plug-in
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This plugin enables Hewlett Packard Enterprise (HPE) Primera (previously 3PAR) storage systems with FiberChannel on KVM hypervisors.
+
+This documentation assumes you have the following configured in your environment before configuring a storage pool in cloudstack:
```

Comment:
The formatting is not proper. Same for Pure.
![Screenshot 2023-12-11 at 10 19 24 AM](https://github.com/apache/cloudstack-documentation/assets/5229147/9d08e568-5b48-4c37-b1d9-a7365dab23fe)